js-weather/script.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+//option要素の取得（配列）
+let options = document.querySelectorAll("#select option");
```

Comment:
```suggestion
const options = document.querySelectorAll("#select option");
```
letである必要がない限り、constで宣言してください。
なおconstで宣言した場合でも変数名はletのときのままでよいです。